lib/package-editor/spec.js

```diff
@@ -1,6 +1,7 @@
 const { expect } = require('chai');
 const packageEditor = require('./');
 const { readJsonSync } = require('fs-extra');
+const { readFileSync } = require('fs');
```

Comment:
[PETTY] You can require it from `fs-extra` as well.

Reply:
It's really not petty, it's a great point. Don't belittle improvements to the codebase